<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicSpectrumWidget</class>
 <widget class="QWidget" name="MusicSpectrumWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>频谱显示</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>418</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>4</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>频谱显示</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>574</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>29</y>
     <width>600</width>
     <height>389</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255, 255, 255, 200);</string>
   </property>
   <widget class="QTabWidget" name="mainViewWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>390</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="spectrumWidget">
     <attribute name="title">
      <string>频谱显示</string>
     </attribute>
     <layout class="QVBoxLayout" name="spectrumLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="spectrumFuncionts" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="spectrumFunciontsLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="analyzerBox">
           <property name="text">
            <string>Analyzer</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ewaveBox">
           <property name="text">
            <string>EWave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="flowwaveBox">
           <property name="text">
            <string>FlowWave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="histogramBox">
           <property name="text">
            <string>Histogram</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="lineBox">
           <property name="text">
            <string>Line</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="spacewaveBox">
           <property name="text">
            <string>SpaceWave</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="spectrumAreaWidget" native="true">
        <layout class="QVBoxLayout" name="spectrumAreaLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="spectrumPlusWidget">
     <attribute name="title">
      <string>高级显示</string>
     </attribute>
     <layout class="QVBoxLayout" name="spectrumPlusLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="spectrumPlusFuncionts" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="spectrumPlusFunciontsLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="foldwaveBox">
           <property name="text">
            <string>FoldWave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="monowaveBox">
           <property name="text">
            <string>Monowave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="multiwaveBox">
           <property name="text">
            <string>Multiwave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="xraysBox">
           <property name="text">
            <string>XRays</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pointxraysBox">
           <property name="text">
            <string>PointXRays</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="volumeWaveBox">
           <property name="text">
            <string>VolumeWave</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="lightenvelopeBox">
           <property name="text">
            <string>LightEnvelope</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="spectrumPlusAreaWidget" native="true">
        <layout class="QVBoxLayout" name="spectrumPlusAreaLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="floridWidget">
     <attribute name="title">
      <string>炫光显示</string>
     </attribute>
     <layout class="QVBoxLayout" name="floridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="floridFuncionts" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="floridFunciontsLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="floridgoomBox">
           <property name="text">
            <string>Goom</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="floridreverbBox">
           <property name="text">
            <string>Reverb</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="floridautismBox">
           <property name="text">
            <string>Autism</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="floridbassBox">
           <property name="text">
            <string>Bass</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="floridsurroundBox">
           <property name="text">
            <string>Surround</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="floridancientBox">
           <property name="text">
            <string>Ancient</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="floridelectricBox">
           <property name="text">
            <string>Electric</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="floridAreaWidget" native="true">
        <layout class="QVBoxLayout" name="floridAreaLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="lightWidget">
     <attribute name="title">
      <string>音质检测</string>
     </attribute>
     <layout class="QVBoxLayout" name="lightLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="lightFuncionts" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="lightFunciontsLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="lightspectrumBox">
           <property name="text">
            <string>LightSpectrum</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="localFileButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>当前歌曲</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="openFileButton">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>其他歌曲</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="lightAreaWidget" native="true">
        <layout class="QVBoxLayout" name="lightAreaLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
